packages/ts-scripts/src/cdk/components/codebuild-project.ts

```diff
@@ -0,0 +1,36 @@
+import * as codebuild from '@aws-cdk/aws-codebuild'
+import * as cdk from '@aws-cdk/core'
+import * as targets from '@aws-cdk/aws-events-targets'
+import { Rule } from '@aws-cdk/aws-events'
+
+import { createSnsTopic, Topic } from './sns-topic'
+
+export const createCodeBuildProject = (stack: cdk.Stack, name: string): codebuild.Project => {
+  const project = new codebuild.Project(stack, name, {
+    buildSpec: codebuild.BuildSpec.fromObject({
+      version: '0.2',
+      phases: {
+        build: {
+          commands: ['echo "No build specified"'],
+        },
+      },
+    }),
+  })
+  
+  return project
+}
+
+export const getCodebuildSnsTopic = (stack: cdk.Stack): Topic => {
+  const topic = createSnsTopic(stack, 'codebuild-sns-topic')
+
+  const rule = new Rule(stack, 'ccodebuild-sns-topic-rule', {
```

Comment:
Made a suggestion but might need to be an option? 
```suggestion
  const rule = new Rule(stack, 'codebuild-sns-topic-rule', {
```

Reply:
there can only be one per stack so it's fine to be hard coded